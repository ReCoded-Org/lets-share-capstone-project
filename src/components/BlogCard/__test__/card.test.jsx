"use strict";

import renderer from "react-test-renderer";

import Card from "../card.jsx";
const item = {
    id: 1,
    title: "My-Blog",
    image: "/Photo.png",
    imageAlt: "Sunset in the mountains",
};

it("renders correctly", () => {
    const tree = renderer.create(<Card item={item} />).toJSON();
    expect(tree).toMatchSnapshot();
});
