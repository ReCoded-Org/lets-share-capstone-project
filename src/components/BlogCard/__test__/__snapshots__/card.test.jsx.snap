// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className=" group max-w-[300px] cursor-pointer overflow-hidden rounded-lg bg-[white] font-primary shadow-xl transition duration-300 hover:scale-[102%] hover:shadow-2xl"
  onClick={[Function]}
  onMouseEnter={[Function]}
>
  <div
    className="relative aspect-video overflow-hidden rounded-md transition-all "
  >
    <span
      style={
        Object {
          "background": "none",
          "border": 0,
          "bottom": 0,
          "boxSizing": "border-box",
          "display": "block",
          "height": "initial",
          "left": 0,
          "margin": 0,
          "opacity": 1,
          "overflow": "hidden",
          "padding": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": "initial",
        }
      }
    >
      <img
        alt="Sunset in the mountains"
        className="abso h-full w-full"
        data-nimg="fill"
        decoding="async"
        onError={[Function]}
        onLoad={[Function]}
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style={
          Object {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": "cover",
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
      <noscript>
        <img
          alt="Sunset in the mountains"
          className="abso h-full w-full"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          sizes="100vw"
          src="/_next/image?url=%2FPhoto.png&w=3840&q=75"
          srcSet="/_next/image?url=%2FPhoto.png&w=640&q=75 640w, /_next/image?url=%2FPhoto.png&w=750&q=75 750w, /_next/image?url=%2FPhoto.png&w=828&q=75 828w, /_next/image?url=%2FPhoto.png&w=1080&q=75 1080w, /_next/image?url=%2FPhoto.png&w=1200&q=75 1200w, /_next/image?url=%2FPhoto.png&w=1920&q=75 1920w, /_next/image?url=%2FPhoto.png&w=2048&q=75 2048w, /_next/image?url=%2FPhoto.png&w=3840&q=75 3840w"
          style={
            Object {
              "border": "none",
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": 0,
              "left": 0,
              "margin": "auto",
              "maxHeight": "100%",
              "maxWidth": "100%",
              "minHeight": "100%",
              "minWidth": "100%",
              "objectFit": "cover",
              "objectPosition": undefined,
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": 0,
            }
          }
        />
      </noscript>
    </span>
    <div
      className=" absolute top-32 h-9 w-24 translate-y-2 bg-primary"
    >
      <p
        className="px-2 py-1 text-sm text-[white]"
      />
    </div>
  </div>
  <div
    className="py-4 px-3 null"
  >
    <h2
      className="text-brand-primary dark:text-white mt-2 text-lg font-semibold tracking-normal"
    >
      <span
        className=" dark:from-purple-800 dark:to-purple-900 bg-gradient-to-r from-[#F07167] to-[#fceceb] bg-[length:0px_10px] bg-left-bottom bg-no-repeat text-lg transition-[background-size] duration-500 hover:bg-[length:100%_3px] group-hover:bg-[length:100%_10px]"
      >
        blogs.headerBlg1
         
      </span>
    </h2>
    <p
      className="text-gray-700 mt-2 text-base "
    >
      blogs.contentBlg1...
    </p>
  </div>
</div>
`;
